src/libstd/sys_common/mod.rs

```diff
@@ -33,8 +33,18 @@ pub mod at_exit_imp;
 #[cfg(feature = "backtrace")]
 pub mod backtrace;
 pub mod condvar;
+#[cfg(any(unix,
+          target_os = "redox",
+          all(target_vendor = "fortanix", target_env = "sgx")))]
+pub mod ffi_bytes;
 pub mod io;
 pub mod mutex;
+#[cfg(any(unix,
```

Comment:
Could this `#[cfg]` annotation perhaps be removed to simply `#![allow(dead_code)]` in the module? It's not really critical we actually delete dead code here and maintaining this `#[cfg]` over time seems like it may get burdensome (especially if it's duplicated with the above)

Reply:
This is not possible because the code doesn't compile as-is on Windows - see https://travis-ci.com/rust-lang/rust/jobs/186485408

I don't think maintaining the cfg (which I put back) is problematic, lots of `sys/` has code like this.